packages/vitest/src/node/core.ts

```diff
@@ -382,6 +389,55 @@ export class Vitest {
     return Promise.all(this.projects.map(w => w.initBrowserProvider()))
   }
 
+  async mergeReports() {
```

Comment:
Should this also handle exit codes when one of the runs failed? E.g. if there's 5 machines running, one of them fails, should the final report merging exit with error code as well?